tutorials/terraform/data-transfer/data-transfer-mmy-objs.tf

```diff
@@ -0,0 +1,160 @@
+# Infrastructure for the Yandex Cloud Data Streams, Object Storage and Data Transfer.
```

Comment:
Infrastructure for the Yandex Cloud Data Streams, Object Storage, and Data Transfer.